examples/fmriprep-docker/README.md

```diff
@@ -63,7 +63,8 @@ Configure the dataset in which you want to collect the results of the (re)comput
 Add a `datalad-remake` special remote:
 
 ```bash
-> git annex initremote datalad-remake type=external externaltype=datalad-remake encryption=none allow-untrusted-execution=true
+> git annex initremote datalad-remake type=external externaltype=datalad-remake \
+encryption=none allow-untrusted-execution=true autoenable=true
 ```
 
```

Comment:
This is not necessary anymore because `datalad make` will automatically start remotes